src/Common/src/CoreLib/Interop/Unix/System.Native/Interop.OpenFlags.cs

```diff
@@ -22,6 +22,7 @@ internal enum OpenFlags
             O_EXCL    = 0x0040,
             O_TRUNC   = 0x0080,
             O_SYNC    = 0x0100,
+            O_DIRECT  = 0x0200,
```

Comment:
Isn't O_DIRECT a Linux-specific value?  How do you plan to support this on macOS?

Reply:
The initial plan was to sneak it into Linux in the same way as is now on Windows, without public API change. Only to avoid hassle with native platform-specific P/Invoke when I really know what I'm doing. Frankly, now it seems easier just to do `fopen` call manually on Linux and give the handle to `FileStream` since it has such ctor overload. macOS support is more involved via fcntl: https://stackoverflow.com/a/2307055/801189